src/lmdb/mdb.c

```diff
@@ -10779,9 +10779,13 @@ mdb_reader_list(MDB_env *env, MDB_msg_func *func, void *ctx)
 	for (i=0; i<rdrs; i++) {
 		if (mr[i].mr_pid) {
 			txnid_t	txnid = mr[i].mr_txnid;
-			snprintf(buf, sizeof(buf), txnid == (txnid_t)-1 ?
-				"%10d %"Z"x -\n" : "%10d %"Z"x %"Yu"\n",
-				(int)mr[i].mr_pid, (size_t)mr[i].mr_tid, txnid);
+                        if (txnid == (txnid_t - 1)) {
+                          snprintf(buf, sizeof(buf), "%10d %"Z"x -\n",
+                                   (int)mr[i].mr_pid);
```

Comment:
`(int)mr[i].mr_pid, (size_t)mr[i].mr_tid);`

OR - if keeping the original file here matters, then the ignore option is 
`-Wno-format-extra-args` on the PKG_CFLAGS line of src/Makevars.win